codice/ddf
# DF-4106 Migrates alert settings & user settings

#### What does this PR do?
 - Migrates alert settings to react, as well as user settings.
 - Updates how we do the plugins for user settings so that they are typed.

#### Who is reviewing it? 
@adimka 
@mdang8 
@pvargas 
@djblue 
@troymohl 

#### How should this be tested?
Go to the alert settings in the Intrigue UI and verify that they work as they did before. 

#### Any background context you want to provide?
This was done as part of a training with @mdang8 and @pvargas to show them the basics of marionette and how to migrate to react.

#### What are the relevant tickets?
[DDF-4106](https://codice.atlassian.net/browse/DDF-4106)

![screen shot 2018-10-29 at 12 07 20 pm](https://user-images.githubusercontent.com/11984853/47673910-357e6d00-db73-11e8-97e7-3359bf3f380c.png)
